dask/tests/test_imports_work.py

```diff
@@ -0,0 +1,17 @@
+import pytest
+
+try:
+    import psutil
+except ImportError:
+    pytest.skip()
+
+
+def test_import_dask_dataframe(monkeypatch):
```

Comment:
Could you move this test over to `dask/dataframe/io/tests/test_csv.py` where we have other related tests (e.g. `test_auto_blocksize`)?